README.md

```diff
@@ -151,11 +158,16 @@ More information about the extension concept can be found here [TBW].
 
 More information about shadowJar can be found [here](https://github.com/johnrengelman/shadow).
 
-## Generate the OpenApi specification
+### Generate the OpenApi specification
```

Comment:
I'm not sure this is something we want to keep in the README, seems something "advanced" to be kept in the developer docs (the `openapi.md` would definitely fit well in the `docs/developer` folder)

Reply:
I would keep this section because this is always a present topic that developer ask for - it doesn't hurt to have it in the `README`. Agree on moving the `openapi.md`.

Reply:
Also added the link to the developer toc